src/sass_types/sass_value_wrapper.h

```diff
@@ -88,6 +92,65 @@ namespace SassTypes
   }
 
   template <class T>
+  NAN_METHOD(SassValueWrapper<T>::Operation) {
+    if (info.Length() < 2) {
+      return Nan::ThrowTypeError("Not enough arguments for operation");
+    }
+    if (!info[0]->IsString()) {
+      return Nan::ThrowTypeError("Operation value should be a string");
+    }
+    if (SassTypes::Value *rhs = SassTypes::Factory::unwrap(info[1])) {
```

Comment:
unwrap doesn't seem to upgrade native type automatically?
I would have wished for it to handle i.e. native numbers transparently for me.
Or is there another utility function that does just that?